server/session/event_log.go

```
@@ -252,45 +252,301 @@ func parseEventsFromReader(reader *bufio.Reader) (parsedEvents, error) {
+	pos := endOffset
+	for pos > 0 {
+		chunk := chunkBytes
+		if chunk > pos {
+			chunk = pos
+		}
+		pos -= chunk
+		tmp := make([]byte, chunk)
+		if _, err := fp.ReadAt(tmp, pos); err != nil && !errors.Is(err, io.EOF) {
+			return SegmentWindow{}, fmt.Errorf("read events file: %w", err)
+		}
+		buffer = append(tmp, buffer...)
```

golangci-lint (2.12.2)

[error] append to slice `tmp` with non-zero initialized length

(makezero)